backend/api/migrations/0002_auto_20170310_2142.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-10 21:42
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import csv
+
+def load_initial_data(apps, schema_editor):
+    Business = apps.get_model("api", "Business")
+    Inspection = apps.get_model("api", "Inspection")
+
+    with open('../../../DOHMH_NYC_Restaurant_Inspection_Results.csv') as csv_file:
+        reader = csv.reader(csv_file, delimiter=',')
+        header = next(reader)
+        for row in reader:
```

Comment:
You don't need to create a separate list for the camises, you can do it all in one list. Check out this gist:

https://gist.github.com/galarant/e056469e7dc4f7f65098d07f21a42358